### PR TITLE
Feature/refactor chain api

### PR DESCRIPTION
**Pull request by ahoseinian**
### Description
Removes the unused code from chainApiContext in order to make it more simple.

### CODE REVIEW

File: src/layout/RegistrarList.tsx [hunk 1]
```diff
@@ -45,14 +45,14 @@ function RegistrarList() {
...
-          renderItem={({item: registrar}) => {
+          renderItem={({item: registrar, index}) => {
             return (
               <View style={styles.registrarTeaserContainer}>
                 <RegistrarTeaser
                   key={registrar.account.toString()}
                   address={registrar.account.toString()}
                   fee={registrar.fee}
-                  index={registrar.index}
+                  index={index}
```
Comment (silva-fj) [inline src/layout/RegistrarList.tsx]: This change will make it depend on the order of the array and not the registrar index itself.. If someone changes the sorting the index will be incorrect

**Review by silva-fj — DISMISSED** [review]
cool 👍🏼 

I just have the concern with the registrar index, but the rest looks good to me

File: src/layout/RegistrarList.tsx (hunk 1 above)
Comment (ahoseinian) [reply src/layout/RegistrarList.tsx]: it was actually not a real index before also, it was the index of the map that was added to the element. https://github.com/litentry/litentry-app/pull/176/files#diff-94b39d441330c0bfb1c8d6c6713ec641959d0aa3425cfe624cce102fe66ef16bL12.
I will just add a todo to look into it later and merge this :)

Comment (silva-fj) [reply src/layout/RegistrarList.tsx]: It is the real registrar index, the thing is that the api does not return the index as a field but in the correct order

Comment (silva-fj) [reply src/layout/RegistrarList.tsx]: We should probably extract that into a function to make it descriptive or add some comments in the meantime

Comment (ahoseinian) [reply src/layout/RegistrarList.tsx]: I've changed it in 19d1b59. please check if you'd like that better.

**Review by silva-fj — APPROVED** [review]

**Pull request merged**